### PR TITLE
Fix for #1019, Enhance parallel obj delete

### PR DESCRIPTION
**Pull request by ramkiperiy**
Enhanced Threading mechanism of deleting objs and added Ignore_left_overs in PVC scale TC's

### CODE REVIEW

**Comment by ramkiperiy** [comment]
Test run logs from laptop. Tear down section
`15:54:45 - MainThread - tests.e2e.test_pv_scale_ocs_create_delete_pvcs - INFO - Created 30 pvc and pods
PASSED                                                                                                                                                                [100%]
----------------------------------------------------------------------------- live log teardown -----------------------------------------------------------------------------
15:54:45 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Pod  -n namespace-test-ce3898128bd44494acf8828f9c931f30 -o yaml
15:54:48 - Thread-136 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-0a48c7986f4c46948256fcc07962c652
15:54:48 - Thread-137 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-0ff5f090414542e59b3bbe21f5673915
15:54:48 - Thread-138 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-241d31f4d81f4ebfbcee48c690c2ae7c
15:54:48 - Thread-139 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-29e1436dbd734c0996f6207c8376b9fc
15:54:48 - Thread-140 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-3486b479f4f040568cef6a71feb1e143
15:54:48 - Thread-141 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-4202cb2d82bc4b85896cbb1464d6b500
15:54:48 - Thread-142 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-4c3e85d161dd45d78532a9d16c362236
15:54:48 - Thread-143 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-6e757e75a08645e9b5826573631efb33
15:54:48 - Thread-144 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-7154152768dc4859ac727c426b836630
15:54:48 - Thread-145 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-7b08ec971787497c847e8eaaa3947a9d
15:54:48 - Thread-146 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-a1de34ab9f674964a430c954f0c17b27
15:54:48 - Thread-147 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-a960277047fb4d44a1e346f548deda52
15:54:48 - Thread-148 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-acbf822684cc49d9bb566e9a7b701e01
15:54:48 - Thread-149 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-b5be90ed06a14b3fbc1b1e655cd3e422
15:54:48 - Thread-150 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-cf168126a85148368d61f5c70f9133c8
15:54:48 - Thread-151 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-e07c5641d0f64536968535c0c5eb5a0e
15:54:48 - Thread-152 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-fd562ce834744d16a5253c831700f389
15:54:48 - Thread-153 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-cephfs-fd56d709316d4555b9b505816167035c
15:54:48 - Thread-154 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-03e3eec8847545159e509907c1ec3a98
15:54:48 - Thread-155 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-0f63fa3f52654f6990a407c01004aae9
15:54:48 - Thread-156 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-15b0c0803d1042c097ac7b5651a394fd
15:54:48 - Thread-157 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-20ddaef228864298aaf619b883c4e7a1
15:54:48 - Thread-158 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-2fb2b51a060a499886c9a04e0b53a3c8
15:54:48 - Thread-159 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-52f98f24a04a4f4c9e89b0fcdbe95ec3
15:54:48 - Thread-160 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-646ddc52768b4a3481f0d8b3044cb2ae
15:54:48 - Thread-161 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-68cf3e59769d4bda8b60ecb3bf3506ad
15:54:48 - Thread-162 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-6dc8e071818741d4bfe64c083dc0de2e
15:54:48 - Thread-163 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-7624845ee1b94cfda457bb46354ff51e
15:54:48 - Thread-164 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-77beb9f0cd7149a0a7db2555b99bb1ec
15:54:48 - Thread-165 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-ab042e408f5c4225bfe0b3b030a99d1a
15:54:49 - Thread-166 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-baa42047f1c74cf79d78f53abd86f9e8
15:54:49 - Thread-167 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-baa7c4edc3704f0d90ebc5c2e61bfdf6
15:54:49 - Thread-168 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-c4d583fbc5064be58170d571fe5a1392
15:54:49 - Thread-169 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-c50d295e3342462cafb350ee91834e30
15:54:49 - Thread-170 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-d89020a0486e4bcbafbc73ff70ca5fd9
15:54:49 - Thread-171 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-e38637c1fabe4765b785a9a2d937f445
15:54:49 - Thread-172 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-f04209c00e374ae38da455ade2feb93d
15:54:49 - Thread-173 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Pod pod-test-rbd-fd2b181b84734d9592ffdb86ebb22d2e
15:55:04 - Thread-174 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-e755f724d78240ce94c8fab73febdb19
15:55:04 - Thread-175 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-78246a3dadf44c4090f1e765155abc35
15:55:04 - Thread-176 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-1eeb00fb36724c71b97be31354517142
15:55:04 - Thread-177 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-f506b099329e4e0c9cee6fc453cbe692
15:55:04 - Thread-178 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-b25e202b51624dd0a68d019ed7a48ea2
15:55:04 - Thread-179 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-f59decc5ff104801a262f9aac1e5fcba
15:55:05 - Thread-180 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-94ba9ed06a6641b9810823aca13e4633
15:55:05 - Thread-181 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-772acf61adf14d6e81620bbf5ff989be
15:55:05 - Thread-182 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-15f18e2b114d478fb8917ad09328cff9
15:55:05 - Thread-183 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-ee626031c65f41dfa46932e3ec5f8808
15:55:05 - Thread-184 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-88f3f554cc2f41e692fe86d791e48afc
15:55:05 - Thread-185 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-a4f53b88522e4eef843e4d7d65f05954
15:55:05 - Thread-186 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-09c18ca11d7247ba8f98bd6151b6a59b
15:55:05 - Thread-187 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-284d0f9a0cdd4bd19fc7f3d8db6e9a2a
15:55:05 - Thread-188 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-c57d403faa3c485db8463ac51c41dc29
15:55:05 - Thread-189 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-597dbf49e58e4440ae724b617789d40c
15:55:05 - Thread-190 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-ff180261f02544dc9c8c30173d12993a
15:55:05 - Thread-191 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-1b1ed0d6465a48919ba1d010fa209ed6
15:55:05 - Thread-192 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-63973ae10593434c860a583cae20ef31
15:55:05 - Thread-193 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-4b158950b5af48eab881971584c0c6a0
15:55:05 - Thread-194 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-4c390c1f54ec4e70a37ca291c1c9045c
15:55:05 - Thread-195 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-eb9f08c8529d4e0693d0e7eb6cdda071
15:55:05 - Thread-196 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-f212fa92affa4d8eaf24154e1f36ba59
15:55:05 - Thread-197 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-c9adf243b6b842efa3e4e29182105af8
15:55:05 - Thread-198 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-c4e178663cc146ec91bdcea4399e782a
15:55:05 - Thread-199 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-c648624efc80466ba755719205feffbc
15:55:05 - Thread-200 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-5ddeae67c7294c0b8e1c7fd2c207846e
15:55:05 - Thread-201 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-2060c9e2c96041deb5e8ded8e2183958
15:55:05 - Thread-202 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-e0219e05b9354596a3d3997850cbe80e
15:55:05 - Thread-203 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-b5032acd2e1b49ada69def867db5833c
15:55:05 - Thread-204 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-7104854385f14d46b9a750c41697f92c
15:55:05 - Thread-205 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-d98fb7baee1d49e887ac05c8184eaec2
15:55:05 - Thread-206 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-b8908a78d7284a7294d6b79fd97753b9
15:55:05 - Thread-207 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-d97b9656797a41848837655eb6668d02
15:55:05 - Thread-208 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-168bede7ec644ddea9877d8722681ddb
15:55:05 - Thread-209 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-34f5e4270eac400cbe2a809d9a3c6376
15:55:05 - Thread-210 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-dbbf2c941db6455b9ec0a717b73645cc
15:55:05 - Thread-211 - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete PersistentVolumeClaim pvc-test-cf86858a1e39436296c9418f110d5e31
15:55:07 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete StorageClass storageclass-test-rbd-6bed06e1ba9b4398932202b3445f1acc
15:55:09 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete StorageClass storageclass-test-cephfs-73b5db92181047d9afb4cc3446a87593
15:55:10 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Attempt to remove resource: storageclass-test-rbd-6bed06e1ba9b4398932202b3445f1acc which isalready deleted! Skipping delete of this resource!
15:55:10 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get StorageClass storageclass-test-rbd-6bed06e1ba9b4398932202b3445f1acc -o yaml
15:55:11 - MainThread - ocs_ci.ocs.ocp - INFO - StorageClass storageclass-test-rbd-6bed06e1ba9b4398932202b3445f1acc got deleted successfully
15:55:11 - MainThread - ocs_ci.ocs.resources.ocs - INFO - Attempt to remove resource: storageclass-test-cephfs-73b5db92181047d9afb4cc3446a87593 which isalready deleted! Skipping delete of this resource!
15:55:11 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get StorageClass storageclass-test-cephfs-73b5db92181047d9afb4cc3446a87593 -o yaml
15:55:12 - MainThread - ocs_ci.ocs.ocp - INFO - StorageClass storageclass-test-cephfs-73b5db92181047d9afb4cc3446a87593 got deleted successfully
15:55:12 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Secret secret-test-rbd-f7e6d8ba70bf42878067492f0b26824e
15:55:14 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Secret secret-test-rbd-f7e6d8ba70bf42878067492f0b26824e -n openshift-storage -o yaml
15:55:15 - MainThread - ocs_ci.ocs.ocp - INFO - Secret secret-test-rbd-f7e6d8ba70bf42878067492f0b26824e got deleted successfully
15:55:15 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Secret secret-test-cephfs-bec1ec6955c64a4e8dafefd869b5bd84
15:55:16 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Secret secret-test-cephfs-bec1ec6955c64a4e8dafefd869b5bd84 -n openshift-storage -o yaml
15:55:18 - MainThread - ocs_ci.ocs.ocp - INFO - Secret secret-test-cephfs-bec1ec6955c64a4e8dafefd869b5bd84 got deleted successfully
15:55:18 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete CephBlockPool cbp-test-166a07c102c14975a51e78a88d084375
15:55:19 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get CephBlockPool cbp-test-166a07c102c14975a51e78a88d084375 -n openshift-storage -o yaml
15:55:21 - MainThread - ocs_ci.ocs.ocp - INFO - CephBlockPool cbp-test-166a07c102c14975a51e78a88d084375 got deleted successfully
15:55:21 - MainThread - ocs_ci.ocs.ocp - INFO - Switching to project openshift-storage
15:55:21 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc project openshift-storage
15:55:22 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig delete Project namespace-test-ce3898128bd44494acf8828f9c931f30
15:55:24 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Project namespace-test-ce3898128bd44494acf8828f9c931f30 -n namespace-test-ce3898128bd44494acf8828f9c931f30 -o yaml
15:55:28 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Project namespace-test-ce3898128bd44494acf8828f9c931f30 -n namespace-test-ce3898128bd44494acf8828f9c931f30 -o yaml
15:55:32 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Project namespace-test-ce3898128bd44494acf8828f9c931f30 -n namespace-test-ce3898128bd44494acf8828f9c931f30 -o yaml
15:55:37 - MainThread - ocs_ci.utility.utils - INFO - Executing command: oc -n namespace-test-ce3898128bd44494acf8828f9c931f30 --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Project namespace-test-ce3898128bd44494acf8828f9c931f30 -n namespace-test-ce3898128bd44494acf8828f9c931f30 -o yaml
15:55:38 - MainThread - ocs_ci.ocs.ocp - INFO - Project namespace-test-ce3898128bd44494acf8828f9c931f30 got deleted successfully
15:55:38 - Dummy-212 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Pod  -A -o yaml
15:55:38 - Dummy-213 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get StorageClass  -A -o yaml
15:55:38 - Dummy-214 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get CephFileSystem  -A -o yaml
15:55:38 - Dummy-215 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get CephBlockPool  -A -o yaml
15:55:38 - Dummy-216 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get PersistentVolume  -A -o yaml
15:55:38 - Dummy-217 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get PersistentVolumeClaim  -A -o yaml
15:55:38 - Dummy-218 - ocs_ci.utility.utils - INFO - Executing command: oc --kubeconfig /home/rperiyas/18thNov/auth/kubeconfig get Namespace  -A -o yaml
`

File: tests/helpers.py [hunk 1]
```diff
@@ -1270,9 +1271,13 @@ def delete_objs_parallel(obj_list):
         bool: True if obj deleted else False
 
     """
-    with ThreadPoolExecutor() as p:
-        for obj in obj_list:
-            p.submit(obj.delete)
+    threads = list()
```
Comment (keesturam) [inline tests/helpers.py]: @ramkiperiy wouldn't this resolve the issue of pvc going into terminating state? why should we use ignore leftover marker?

Comment (ramkiperiy) [reply tests/helpers.py]: During the testing we have ceph pod re-spin, due to this also observing leftovers.

**Comment by ramkiperiy** [comment]
@vasukulkarni Can you please review and merge this PR

**Pull request merged**